doc/source/conf.py

```diff
@@ -237,9 +251,10 @@
     "backreferences_dir": None,
     # Modules for which function level galleries are created.  In
     "doc_module": "ansys-mapdl-core",
-    "image_scrapers": ("pyvista", "matplotlib"),
-    "ignore_pattern": "flycheck*",
-    "thumbnail_size": (350, 350),
+    "image_scrapers": (DynamicScraper(), "matplotlib"),
+    # "ignore_pattern": "flycheck*",
+    # "thumbnail_size": (350, 350),
+    "reset_modules_order": "both",
```

Comment:
Any reason for those commented lines?
If you need to comment them, please do delete them.

Reply:
This was used for development purpose. It will be removed.